frontend/src/views/Collection/index.tsx

```diff
@@ -1,14 +1,114 @@
+import { Button, Classes, H3, H4, Intent, UL } from "@blueprintjs/core";
+import { IconNames } from "@blueprintjs/icons";
 import { RouteComponentProps } from "@reach/router";
+import psl from "psl";
 import React, { FC } from "react";
+import { Link, LINK_TYPE } from "src/common/entities";
+import { useCollection } from "src/common/queries/collections";
+import {
+  CenterAlignedDiv,
+  CollectionButtons,
+  CollectionInfo,
+  DatasetContainer,
+  Description,
+  LinkContainer,
+  StyledButton,
+  ViewGrid,
+} from "./style";
 
 interface RouteProps {
-  id: string;
+  id?: string;
 }
 
 export type Props = RouteComponentProps<RouteProps>;
 
+const getDomain = (url: string): string | null => {
```

Comment:
is it possible to use `new URL` instead of `psl`? Just wondering if that'd save us extra JS weight 😄 Thank you!